src/dcurl.c

```diff
@@ -33,9 +33,21 @@
 
 /* check whether dcurl is initialized */
 static bool isInitialized = false;
+static uint8_t initStatus = 0;
 static uv_sem_t notify;
 
+/* for checking whether the corresponding implementation is initialized */
+enum Capability {
+    CAP_C = 1U,
```

Comment:
If new entry `CAP_NONE = 0` is specified, `initStatus` can be set to `CAP_NONE`, that would be more consistent.

Reply:
Agree. It would be more readable.

Reply:
Done.